### PR TITLE
Actions/build backend

### PR DESCRIPTION
**Pull request by ErlingHauan**
Added a Github Action for automatically building and testing the backend when new pushes are made to the /backend directory.

### CODE REVIEW

File: .github/workflows/backend-build-test.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          dotnet-version: '8.0.x'
+
+      - name: Restore dependencies
+        run: dotnet restore backend/FormAPI.sln
+
+      - name: Build
+        run: dotnet build backend/FormAPI.sln --no-restore
+        
+      - name: Test
+        run: dotnet test backend/FormAPI.sln --no-build
```
Comment (standeren) [inline .github/workflows/backend-build-test.yml:26]: Same comment as on the other PR - might consider splitting these steps into two jobs for easier monitoring and analyzing if failure 🫣

File: .github/workflows/backend-build-test.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '8.0.x'
```
Comment (standeren) [inline .github/workflows/backend-build-test.yml:17]: Small nitpicking, but looks cleaner if the quotations is consequentially used as singular og double 😛
```suggestion
          dotnet-version: "8.0.x"
```

File: .github/workflows/backend-build-test.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+on:
+  push:
+    paths:
+      - "backend/**"
+
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```
Comment (standeren) [inline .github/workflows/backend-build-test.yml:12]: Should this step also have a name?

**Review by standeren — APPROVED** [review]
Looks good! 
One thing you could consider is introducing `dotnet format` command fix whitespaces and code styling.

File: .github/workflows/backend-build-test.yml (hunk 2 above)
Comment (ErlingHauan) [reply .github/workflows/backend-build-test.yml]: Hard agree!

**Pull request merged**